src/Player.java

```diff
@@ -104,4 +100,22 @@ public int getRations() {
     public void setRations(int rations) {
         this.rations = rations;
     }
+
+    public int getAliveOxenCount(){
+        int count = 0;
+        for(Entity e: oxen) count += e.isAlive() ? 1: 0;
+        return count;
+    }
+
+    public int getTotalOxenHealth(){
+        int health = 0;
+        for(Entity e: oxen) health += e.health;
+        return health;
+    }
+
+    public int getTravelDistance(){
```

Comment:
I think this looks great, but we will need to limit the amount of Oxen the player can buy as a future issue